src/toast/observation.py

```diff
@@ -68,6 +68,7 @@ def set_default_values(values=None):
         "shared_mask_irregular": 4,
         "det_mask_invalid": 1,
         "det_mask_sso": 1 + 2,
+        "det_mask_processing": 2,
```

Comment:
This is fine, although we could consider renaming this to `det_mask_filter` if that is what the intended use is rather than the generic "processing".

Reply:
The flag bits could be used in destriping as well so I picked this general name.